Core/Types/CkanModule.cs

```diff
@@ -366,6 +376,33 @@ private void DeSerialisationFixes(StreamingContext like_i_could_care)
             name = name ?? string.Empty;
         }
 
+        static CkanModule()
+        {
+            var assembly = Assembly.GetExecutingAssembly();
+            using (var stream = assembly.GetManifestResourceStream(SchemaPath))
+            using (var reader = new StreamReader(stream))
+            {
+                var result = reader.ReadToEnd();
+                MetadataSchema = JsonSchema4.FromSampleJson(result);
+            }
+        }
+
+        private static readonly string SchemaPath = "CKAN.CKAN.schema";
```

Comment:
Let's make this a `const` instead of `static readonly`, since the value is known at compile time.